scipy/_lib/tests/test__util.py

```diff
@@ -331,6 +332,33 @@ def test_contains_nan_2d(self):
         data4 = np.array([["1", 2], [3, np.nan]], dtype='object')
         assert _contains_nan(data4)[0]
 
+    @array_api_compatible
+    @pytest.mark.parametrize("nan_policy", ['propagate', 'omit', 'raise'])
+    def test_array_api(self, xp, nan_policy):
```

Comment:
is this similar to the other PR in that this test is here in place of pending improvements to the other tests, or is it here to stay?

Reply:
Similar to `pearsonr`.